scenarios/kubernetes_e2e.py

```diff
@@ -377,6 +382,11 @@ def main(args):
             ' --kubernetes-anywhere-phase2-provider kubeadm' \
             ' --kubernetes-anywhere-cluster %s' \
             ' --kubernetes-anywhere-kubeadm-version %s' % (cluster, version)
+
+        k8s_version = mode.job_env().get('KUBEADM_KUBERNETES_VERSION')
```

Comment:
docker mode does not have `job_env()` though... you might want to check mutual exclusive?